docs/config.rst

```diff
@@ -240,6 +240,19 @@ Configuration Options
   * Default: "" (Jetty's default)
   * Importance: medium
 
+``kafkastore.bootstrap.servers``
```

Comment:
This file is generated from the `SchemaRegistryConfig` class. I think a lot of the extra text that's here and not in `KAFKASTORE_BOOTSTRAP_SERVERS_DOC` probably covers my earlier comments.